packages/graphql-transformer-core/src/util/fileUtils.ts

```diff
@@ -1,79 +1,35 @@
-import * as fs from 'fs';
 import * as path from 'path';
-
-/**
- * Promisify common operations.
- */
-
-export const readDir = async (dir: string) => await promisify<string, string[]>(fs.readdir, dir)
-export const readFile = async (p: string) => await promisify(fs.readFile, p)
-export const writeFile = async (p: string, d: any, opts: any = {}) => await new Promise((res) => fs.writeFile(p, d, opts, e => res(e)))
-export const lstat = async (dir: string) => await promisify(fs.lstat, dir)
-export const mkdir = async (p: string) => await new Promise((res) => fs.mkdir(p, e => res(e)))
-export const exists = async (p: string) => await new Promise((res) => fs.exists(p, e => res(e)))
-export const unlink = async (p: string) => await new Promise((res, rej) => fs.unlink(p, e => e ? rej(e) : res()))
-export const rmdir = async (p: string) => await new Promise((res, rej) => fs.rmdir(p, e => e ? rej(e) : res()))
-function promisify<A, O>(fn: (arg: A, cb: (err: Error, data: O) => void) => void, a: A): Promise<O> {
-    return new Promise((res, rej) => {
-        fn(a, (err, d) => {
-            err ? rej(err) : res(d)
-        })
-    })
-}
+const fs = require('fs-extra');
 
 /**
  * Helpers
  */
 
-export async function deleteDirectory(directory: string): Promise<void> {
-    const pathExists = await exists(directory);
-    if (!pathExists) {
-        return;
-    }
-    await emptyDirectory(directory);
-    await rmdir(directory);
-}
-
 export async function emptyDirectory(directory: string) {
```

Comment:
may be we can directly use [emptyDir](https://github.com/jprichardson/node-fs-extra/blob/master/docs/emptyDir.md)